docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -36,12 +36,10 @@ but a non-SGX machine can still be used:
 
 On the Linux build machine, or after opening an ssh session into the VM:
 
-- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 16.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md)
-  or [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
+- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
   in SGX not working.  Instead, replace step 2 with the
```

Comment:
Hi @radhikaj, are the instructions in lines 40-42 still relavent?

Reply:
@rs-- Can you please help us understand if 40-42 are still relevant?

Reply:
You can remove: "except that step 2 on those pages is outdated and result in SGX not working. Instead, replace step 2 with the SGX1 instructions prior to the Install section."